app/controllers/students_controller.rb

```diff
@@ -1,36 +1,17 @@
 class StudentsController < ApplicationController
   include ApplicationHelper
 
-  rescue_from Exceptions::EducatorNotAuthorized, with: :redirect_unauthorized!
-
-  before_action :authorize!, except: [          # The :names and lasids actions are subject to
-    :names, :lasids                             # educator authentication via :authenticate_educator!
-  ]                                             # inherited from ApplicationController.
-                                                # They then get further authorization filtering using
-                                                # The custom authorization methods below.
-
-  before_action :authorize_for_districtwide_access_admin, only: [:lasids]
-
-  def authorize!
-    student = Student.find(params[:id])
-    raise Exceptions::EducatorNotAuthorized unless current_educator.is_authorized_for_student(student)
-  end
-
-  def authorize_for_districtwide_access_admin
-    unless current_educator.admin? && current_educator.districtwide_access?
-      render json: { error: "You don't have the correct authorization." }
-    end
-  end
```

Comment:
So this is one of the simplification benefits - this code gets replaced with wrapping `authorized` around data being loaded in the controller methods.